docs/learn/learn-comparison-avalanche.md

```diff
@@ -0,0 +1,104 @@
+
+## Staking Mechanics
+
+<!-- TODO: Add Polkadot comparison to become a validator and nominator -->
+
+Avalanche uses a Delegated Proof-of-Stake mechanism without any slashing. The barrier to entry for staking as a full node validator is 2500 AVAX, and 25 AVAX to become a delegator. 
+
+## Shards
+
+<!-- TODO: Add sharding comparison... this will mainly be subnets vs parachains -->
+
+## Message Passing
+
+Avalanche does not have a native trustless message passing mechanism, instead it relies on bridges. Though, because it is an EVM compatible protocol, it's able to interoperate at a token level, however subnets do not have a messaging layer out of the box. Polkadot, with its XCM and XCMP messaging protocols allows for a native and trustless messaging scheme, making composibility of chains to build powerful applications much simpler.
+
+## Governance
+<!-- 
+TODO: Add definitions to each key parameter that is subject to change via governance -->
+
+Avalanche plans to have an on-chain governance mechanism based on its whitepaper. Even though it does not currently have an on-chain or off-chain system in production. Its governance system will be subject to only a few key protocol parameters which include:
+
+- Staking amount
+- Minimum staking time for a node
+- Maximum staking time for a node
+- Minting rate
+- Transaction fee amount
+
+Limiting the governance functionality is a design choice to increase predictibility and safety.
+
+Polkadot's governance mechanism has been in production from the very begining, and was used to slowly release functionality and decentralize the initial network. It is also not limited to a few parameters, the whole runtime is subject to change via protocol.
+
+## Upgrades
+
+The upgradibility of Avalanche is up to the protocol devleopers at Avax Labs vs the forkless upgrade funcitonality of Polkadot that is achieved through on-chain governance.
+
+## Conclusion
+
+In concolusion, Avalanche has made some design decisions which allow for an improved smart-contract development environment in which protocol engineers can have the freedom to create their own blockchains and include them into the Avalanche ecosystem via subnets. The trade-offs being that autonomy of design is limited and blockchains have to buy into the design decisions of Avalanches main chains. Subnets are not able to share the security of the main chains however they are able to use the low-transaction fee to create working applicaiton quick. We have even seen some Ethereum projects move to Avalanche to take advantage of the lower fees. Whereas Polkadot provides a heteregeneous model, in which blockchains have ultimate autonomy in the way they design their consensus systems, and on top of that are able to share the security of the main Relay-chain, and use XCM to pass native trustless messages instead of having to rely on bridges.
```

Comment:
" However, they are able to utilize the low-transaction fee to create working applications quick. " might need rephrasing. Transaction fee is not a blocker for building and deploying working applications

```suggestion
In conclusion, Avalanche has made some design decisions which allow for an improved smart-contract development environment in which protocol engineers can have the freedom to create their own blockchains and include them into the Avalanche ecosystem via subnets. The trade-offs are that the autonomy of design is limited and blockchains have to buy into the design decisions of Avalanche's main chains. Subnets are not able to share the security of the main chains. However, they are able to utilize the low-transaction fee to create working applications quick. We have even seen some Ethereum projects move to Avalanche to take advantage of the lower fees. Whereas Polkadot provides a heterogeneous model, in which blockchains have ultimate autonomy in the way they design their consensus systems, and on top of that are able to share the security of the Relay-chain, and use XCM to pass native trustless messages, instead of having to rely on bridges.
```